app/services/search/search_builder.rb

```diff
@@ -49,7 +48,11 @@ def build_search_replica
   end
 
   def build_suggestions
-    @wider_radius_suggestions = Search::SuggestionsBuilder.new(search_params, @params_hash[:radius]).radius_suggestions
+    @wider_radius_suggestions = if @point_coordinates.present?
```

Comment:
I wonder if having `radius` in the variable name will be confusing, because we are referring to radiuses and buffers differently in the code. How about `wider_search_suggestions`?

Reply:
I think that makes sense, yeah!